plans/main.fmf

```diff
@@ -2,15 +2,66 @@
 /:
     inherit: false
 
+discover:
+    how: fmf
+
+execute:
+    how: tmt
+
 adjust:
-  - prepare:
+  - prepare+:
```

Comment:
Why `+`? It doesn't inherit from top level, leaving `prepare:` would make it clean for reader no additional prepare steps coming from top.

Reply:
I remember thinking the same and testing it, and it removed the other `prepare` content.

What I think is happening is that `adjust` is being applied **on top** of regular dictionary keys, so if `/plans/foobar.fmf` specifies `prepare:`, it gets used first, and then all the `adjust`s are applied, starting from the top-level `/plans/main.fmf`.